app/grants/clr.py

```diff
@@ -394,10 +421,42 @@ def predict_clr(save_to_db=False, from_date=None, clr_round=None, network='mainn
     v_threshold = float(clr_round.verified_threshold)
     uv_threshold = float(clr_round.unverified_threshold)
 
+    print(total_pot)
+
     grants, contributions, phantom_funding_profiles = fetch_data(clr_round, network)
 
     grant_contributions_curr = populate_data_for_clr(grants, contributions, phantom_funding_profiles, clr_round)
 
+    # pre-calculate clr amount to figure out a closer total pot
+    pre_pot = 0
+    for grant in grants:
+        p_clr, g_clr, _, _ = calculate_clr_for_donation(
+                grant,
+                0,
+                grant_contributions_curr,
+                total_pot,
+                v_threshold,
+                uv_threshold
+            )
+        
+        # print(f'{grant}\namount: {p_clr}')
+       
+        if p_clr is None: 
+            pre_pot += 0
+        if p_clr is not None:
+            pre_pot += p_clr
+
+    print(f'match hit so far using full pot: {pre_pot}')
+
+    # using clr amount, round total pot up to the closest 10k
+    # I think this needs to be changed to round down
+    total_pot_close = round(pre_pot + 5000, -4) 
+
+    print(f'using {total_pot_close} to calculate preds based on full pot') 
+
+    # # when not using our pot mechanisms
+    # total_pot_close = total_pot
```

Comment:
What are you thoughts on shifting this over to
https://github.com/gitcoinco/web/blob/master/app/grants/clr.py#L208

That is where our normalization code resides !
Modifying the total_pot to use `total_pot_close` would mess up the value of `CLR_PERCENTAGE_DISTRIBUTED`

We could do something like

```
    if bigtot >= total_pot: # saturation reached
        # print(f'saturation reached. Total Pot: ${total_pot} | Total Allocated ${bigtot}. Normalizing')
        CLR_PERCENTAGE_DISTRIBUTED = 100
        for t in totals:
            t['clr_amount'] = ((t['clr_amount'] / bigtot) * total_pot)
    else:
        CLR_PERCENTAGE_DISTRIBUTED =  (bigtot / total_pot) * 100
        # your code starts here
        where you can define a `temp_total_pot` is X% higher than `bigtot`
       and then do something like 
        for t in totals:
            t['clr_amount'] = ((t['clr_amount'] / bigtot) * temp_total_pot)


```